src/app/api/v1/user/update-comm-option/route.ts

```diff
@@ -11,28 +11,45 @@ import AppConstants from "../../../../../appConstants";
 import {
   getSubscriberByFxaUid,
   setAllEmailsToPrimary,
+  setMonthlyMonitorReport,
 } from "../../../../../db/tables/subscribers";
 
+export type EmailUpdateCommTypeOfOptions = "null" | "affected" | "primary";
+
 export interface EmailUpdateCommOptionRequest {
-  /** "1" to send breach alerts to the primary email address, "0" to send them to the affected address */
-  communicationOption: "0" | "1";
+  instantBreachAlerts: EmailUpdateCommTypeOfOptions;
+  monthlyMonitorReport: boolean;
 }
 
 export async function POST(req: NextRequest) {
   const token = await getToken({ req });
 
   if (typeof token?.subscriber?.fxa_uid === "string") {
     try {
-      const { communicationOption }: EmailUpdateCommOptionRequest =
-        await req.json();
+      const {
+        instantBreachAlerts,
+        monthlyMonitorReport,
+      }: EmailUpdateCommOptionRequest = await req.json();
       const subscriber = await getSubscriberByFxaUid(token.subscriber?.fxa_uid);
       if (!subscriber) {
         throw new Error("No subscriber found for current session.");
       }
-      // 0 = Send breach alerts to the corresponding affected emails.
-      // 1 = Send all breach alerts to user's primary email address.
-      const allEmailsToPrimary = Number(communicationOption) === 1 ?? false;
+      // "null"     = Do not send instant notifications. Newly added in MNTOR-1368
+      // "affected" = Send breach alerts to the corresponding affected emails.
+      // "primary"  = Send all breach alerts to user's primary email address.
+      let allEmailsToPrimary;
```

Comment:
tiny (and easily ignorable) nit: Change this to `null` default and drop the `switch..default` case below?
```suggestion
      let allEmailsToPrimary = null;
```